machine/machine_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func loadFixtrure(file string, t *testing.T) *StateMachine {
+func loadFixture(file string, t *testing.T) *StateMachine {
```

Comment:
I think you misspelled `Fixture` here, it is meant to be spelled `Fixtrure` /s :) +1